proto/viam/service/slam/v1/slam.proto

```diff
@@ -79,8 +79,8 @@ message GetPointCloudMapResponse {
   // format where XY is the ground plane and positive Z is up, following
   // the Right Hand Rule.
   //
-  // Read more about the pointcloud format here:
-  // https://pointclouds.org/documentation/tutorials/pcd_file_format.html
```

Comment:
only thought is if we should use something else besides "in the docs" since the link doesn't go to our docs. maybe "in the pcd file documentation"?

Reply:
I'm fine with that, and actually originally wrote it that way. But that exceeds the 80 character limit that this file otherwise respects, and if I split up the markdown link between two separate lines, I'm afraid it'll wind up getting broken downstream since the PYI files preserve whitespace by inserting manual (double) newlines directly into their docstrings.

Choose a door, @JohnN193 :

a) violate the 80 character limit?
b) risk a broken link downstream that is both ugly and doesn't fix the issue
c) leave my admittedly sparse link text?

Reply:
@JohnN193 Is this OK to merge, or would you rather we make a change? Thanks!